src/server/routes/editor.js

```diff
@@ -240,29 +242,7 @@ router.get('/:id/revisions', (req, res, next) => {
 				}
 			}
 		})
-		.then((editor) => {
-			const editorJSON = editor.toJSON();
-			let editorTitleJSON;
-			if (editorJSON.titleUnlockId === null) {
-				editorTitleJSON = Promise.resolve(editorJSON);
-			}
-			else {
-				editorTitleJSON = new TitleUnlock({
-					id: editorJSON.titleUnlockId
-				})
-					.fetch({
-						withRelated: ['title']
-					})
-					.then((unlock) => {
-						if (unlock !== null) {
-							editorJSON.title =
-								unlock.relations.title.attributes;
-						}
-						return editorJSON;
-					});
-			}
-			return editorTitleJSON;
-		})
+		.then(getEditorTitleJSON, TitleUnlock)
```

Comment:
Editor is being passed in here, not JSON - need to convert, I think.

Reply:
Same as above.